platform/persistence/platform-persistence-core-api/src/main/java/org/codice/ddf/persistence/PersistentStore.java

```diff
@@ -102,5 +111,5 @@ public String toString() {
    * @return Count of the items deleted
    * @throws PersistenceException
    */
-  public int delete(String type, String ecql) throws PersistenceException;
+  int delete(String type, String ecql) throws PersistenceException;
```

Comment:
The implementation of delete calls the original get method so delete is limited to 10 results still. Should probably address that in this pr as well.

Reply:
Added